src/lib/y2partitioner/widgets/summary_text.rb

```diff
@@ -83,8 +83,11 @@ def refresh_value
       end
 
       # Updates the value of {#actions}
+      #
+      # @note Actions are obtained from the raw probed devicegraph
```

Comment:
Not sure if this sentence is clear enough. It can look like you ask the list of actions to raw_probed instead of current_graph.  I would say something like "Actions are calculated with the raw probed devicegraph as starting point".